lib/admin.php

```diff
@@ -236,7 +246,7 @@ function wc_admin_enqueue_script() {
 add_action( 'admin_enqueue_scripts', 'wc_admin_enqueue_script' );
 
 /**
- * Adds an admin body class.
+ * Adds body classes to the main wp-admin wrapper, allowing us to better target elements in specific scenarios.
```

Comment:
Really, really like this. This is something I wish was present in calypsoify too.